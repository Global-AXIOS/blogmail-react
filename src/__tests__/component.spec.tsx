import * as React from "react"

import Blogmail from ".."
import renderer from "react-test-renderer"

test("Component should show 'red' text 'Hello World'", () => {
  const component = renderer.create(<Blogmail newsletterId="test" />)
  const testInstance = component.root

  expect(testInstance.findByType(Blogmail).props.newsletterId).toBe("test")

  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
