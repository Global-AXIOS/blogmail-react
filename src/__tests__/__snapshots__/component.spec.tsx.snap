// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should show 'red' text 'Hello World' 1`] = `
<div>
  <div
    id="blogmail1"
  >
    <script
      dangerouslySetInnerHTML={
        Object {
          "__html": "{\\"newsletterId\\":\\"test\\"}",
        }
      }
      type="application/json"
    />
  </div>
</div>
`;
